components/brave_wallet_ui/common/hooks/transaction-parser.ts

```diff
@@ -87,12 +87,13 @@ export function useTransactionFeesParser (selectedNetwork: BraveWallet.NetworkIn
   }, [])
 
   return React.useCallback((transactionInfo: BraveWallet.TransactionInfo): ParsedTransactionFees => {
-    const { txDataUnion: { ethTxData1559: txData }, txType } = transactionInfo
+    const { txDataUnion: { ethTxData1559: txData, filTxData }, txType } = transactionInfo
```

Comment:
this is just to be able to test, the rest will be done in https://github.com/brave/brave-browser/issues/22020